examples/todo/js/mutations/AddTodoMutation.js

```diff
@@ -42,12 +42,15 @@ export default class AddTodoMutation extends Relay.Mutation {
       parentID: this.props.viewer.id,
       connectionName: 'todos',
       edgeName: 'todoEdge',
-      rangeBehaviors: {
-        '': 'append',
-        'status(any)': 'append',
-        'status(active)': 'append',
-        'status(completed)': null,
-      },
+      rangeBehaviors: ({status}) => {
+        if (status === 'any' || status === 'active') {
+          return 'append';
+        } else if (status === 'completed') {
+          return null;
```

Comment:
Totally :) this PR is 4 months old but I would gladly rebase and modify it if this is still something you wanted to introduce! What do you think?